### PR TITLE
fix(image-card): display preview not available error for collection variant

### PR DESCRIPTION
**Pull request by NazimHAli**
**This change:** (check at least one)

- [ ] Adds a new feature
- [x] Fixes a bug
- [ ] Improves maintainability
- [ ] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [x] No

**Is the:** (complete all)

- [x] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [x] Test suite(s) passing?
- [ ] Code coverage maximal?
- [ ] Changeset added?
- [ ] Component status page up to date?

**What does this change address?**

The preview not available element doesn't render for image errors with the `collection` variant.

Example error with the [change preview](https://60919c26122bd50039b34644-sezdfjtplq.chromatic.com/?path=/story/components-image-card--collection)

Screenshot 2023-05-03 at 5 59 42 PM

### CODE REVIEW

File: packages/pharos/src/components/image-card/pharos-image-card.scss [hunk 1]
```diff
@@ -168,6 +168,23 @@ slot[name='image']::slotted(img) {
   height: auto;
 }
 
+.card__link--collection--error {
+  background-color: var(--pharos-color-marble-gray-94);
+  height: 11.25rem;
+  width: 15rem;
+  display: flex;
+  flex-direction: column;
```
Comment (NazimHAli) [inline packages/pharos/src/components/image-card/pharos-image-card.scss:176]: Copied most of this from the other error class, but don't like the hardcoded dimensions. Going to work on updating it to be responsive.

**Review by chrisjbrown — APPROVED** [review]
nice!

File: packages/pharos/src/components/image-card/pharos-image-card.scss [hunk 2]
```diff
@@ -165,7 +165,28 @@ slot[name='image']::slotted(img) {
 .card__link--collection {
   display: grid;
   position: relative;
-  height: auto;
+  height: 14rem;
```
Comment (NazimHAli) [inline packages/pharos/src/components/image-card/pharos-image-card.scss:168]: Updated the `height` to match the `height` set with image link and container error. Setting the height also fixes the mismatched heights when its set to `auto` https://github.com/ithaka/pharos/blob/develop/packages/pharos/src/components/image-card/pharos-image-card.scss#L23

Comment (daneah) [reply packages/pharos/src/components/image-card/pharos-image-card.scss:168]: @NazimHAli it seems like this potentially impacts visual/editorial design fairly significantly, comparing how much of the image covers the card area. Is there a sense from design whether this is alright?

Comment (NazimHAli) [reply packages/pharos/src/components/image-card/pharos-image-card.scss]: I noticed this too while reviewing collection variant in prod vs this, testing it with the same height here https://github.com/ithaka/pharos/pull/516/commits/5000badd5f55110a1d49d7ebb19f89aca6fee28a

**Review by chrisjbrown — APPROVED** [review]

**Review by jialin-he — APPROVED** [review]

**Review by daneah — APPROVED** [review]

**Pull request merged**